### PR TITLE
Documentation string

### PR DESCRIPTION
**Pull request by HansOlsson**
Use description-string and annotation-clause in the grammar (as decided).

### CODE REVIEW

File: chapters/syntax.tex [hunk 1]
```diff
@@ -55,7 +55,7 @@ \section{Lexical conventions}\doublelabel{lexical-conventions}
   has structured comments in the form of annotations and string
   comments.
 \item
-  Each description-string or string in annotations (= STRING with production annotation-comment in the
+  Each description-string or string in annotations (= STRING with production annotation-clause in the
```
Comment (svorkoetter) [inline chapters/syntax.tex]: It's not clear to me what, "(= STRING with production annotation-clause in the grammar)", actually means.

Comment (HansOlsson) [reply chapters/syntax.tex]: I also noticed that when looking through the text. My current guess is that it should be "within" instead of "with".

**Comment by svorkoetter** [comment]
Should we change that, perhaps as a separate PR?

On Fri, 20 Dec 2019 07:03:15 -0800
Hans Olsson <notifications@github.com> wrote:

> HansOlsson commented on this pull request.
> 
> 
> 
> > @@ -55,7 +55,7 @@ \section{Lexical conventions}\doublelabel
> > {lexical-conventions}
>    has structured comments in the form of annotations and string
>    comments.
>  \item
> -  Each description-string or string in annotations (= STRING with
> production annotation-comment in the
> +  Each description-string or string in annotations (= STRING with
> production annotation-clause in the
> 
> I also noticed that when looking through the text. My current guess
> is that it should be "within" instead of "with".
> 
> -- 
> You are receiving this because your review was requested.
> Reply to this email directly or view it on GitHub:
> https://github.com/modelica/ModelicaSpecification/pull/2467#discussion_r360410424


-- 
Stefan Vorkoetter
Senior Architect
Maplesoft

**Comment by HansOlsson** [comment]
> Should we change that, perhaps as a separate PR?
> …

Yes.

**Pull request merged**